modules/client-configuration/pages/contact-methods-saltssh.adoc

```diff
@@ -6,6 +6,13 @@ Push via Salt SSH is used in environments where Salt clients cannot reach the {p
 In this environment, clients are located in a firewall-protected zone called a DMZ.
 No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
 
+[IMPORTANT]
+====
+If you want to use Push via Salt SSH you must avoid installing the Salt bundle on the client.
+For more information about the Salt bundle, see xref:client-configuration:registration-methods-saltbundle.adoc[].
+====
+
+
```

Comment:
I think it was a misunderstanding the idea of the relation Salt Bundle to Salt-SSH. Let's just remove this section.

Reply:
ok, done.